dotCMS/src/main/java/com/dotcms/rest/exception/LicenseRequiredException.java

```diff
@@ -0,0 +1,20 @@
+package com.dotcms.rest.exception;
+
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+
+/**
+ * Throw when feature is EE only
+ */
+public class LicenseRequiredException extends HttpStatusCodeException {
```

Comment:
We already have a `InvalidLicenseException`, we should use it instead of create a new Exception as we already discussed with Freddy R.

Reply:
ready